sdk/spring-sdk/src/test/scala/kalix/springsdk/impl/ViewDescriptorFactorySpec.scala

```diff
@@ -16,15 +16,20 @@
 
 package kalix.springsdk.impl
 
-import kalix.springsdk.testmodels.view.ViewTestModels.TransformedUserView
-import kalix.springsdk.testmodels.view.ViewTestModels.UserByEmailWithGet
-import kalix.springsdk.testmodels.view.ViewTestModels.UserByEmailWithPost
-import kalix.springsdk.testmodels.view.ViewTestModels.UserByEmailWithPostRequestBodyOnly
-import kalix.springsdk.testmodels.view.ViewTestModels.UserByNameEmailWithPost
-import kalix.springsdk.testmodels.view.ViewTestModels.UserByNameStreamed
-import kalix.springsdk.testmodels.view.ViewTestModels.ViewWithNoQuery
-import kalix.springsdk.testmodels.view.ViewTestModels.ViewWithSubscriptionsInMixedLevels
-import kalix.springsdk.testmodels.view.ViewTestModels.ViewWithTwoQueries
+import kalix.springsdk.testmodels.view.ViewTestModels.{
```

Comment:
format?

Reply:
fixed now, although I think we should probably rely this to the scala fmt